src/Pages/Character/index.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import ProductList from '../../Components/ProductList';
+import { API } from '../../config';
+import { categoryData } from '../../Data/categoryData';
+import { characterData } from '../../Data/characterData';
+import { fetchGet } from '../../utils/fetches';
+
+import FilterModal from './FilterModal';
+import './index.scss';
+
+class Character extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isOpen: false,
```

Comment:
```suggestion
      isModalOpen: false,
```
어떤 대상이 열렸는지 변수명에 표시해주는 것이 가독성 측면에서 좋을 거 같아요

Reply:
수정했습니다.